x-pack/plugins/aiops/common/api/log_categorization/schema.ts

```diff
@@ -6,6 +6,39 @@
  */
 
 import { schema, TypeOf } from '@kbn/config-schema';
+import { i18n } from '@kbn/i18n';
+import { isRuntimeField } from '@kbn/ml-runtime-field-utils';
+
+export const runtimeMappingsSchema = schema.object(
```

Comment:
Wonder if we should add runtimeMappingsSchema to `ml-runtime-field-utils` as we do use it in several plugins 🤔

Reply:
Moved in https://github.com/elastic/kibana/pull/162319/commits/40fb37026d60c7f10f02231c0dd3a9c7bd84f462

Reply:
I've reverted this move. I caused a 200KB bundle increase in ML and Transforms.